src/Og.php

```diff
@@ -96,21 +102,35 @@ public static function createField($plugin_id, $entity_type, $bundle, array $set
         'targetEntityType' => $entity_type,
         'bundle' => $bundle,
         'mode' => 'default',
-        'status' => TRUE,
       ]);
     }
 
-    $widget = $form_display->getComponent($plugin_id);
-    $widget['type'] = 'og_complex';
-    $widget['settings'] = [
-      'match_operator' => 'CONTAINS',
-      'size' => 60,
-      'placeholder' => '',
-    ];
+    $form_display_definition = $og_field->getFormDisplayDefinition($settings['form_display']);
+
 
-    $form_display->setComponent($plugin_id, $widget);
+    $form_display->setComponent($plugin_id, $form_display_definition);
     $form_display->save();
 
+
+    // Set the view display for the "default" and "teaser" types.
+    $view_display_definition = $og_field->getViewDisplayDefinition($settings['view_display']);
+
+    foreach (['default', 'teaser'] as $mode) {
+
+      /** @var EntityDisplayInterface $view_display */
+      if (!$view_display = \Drupal::entityTypeManager()->getStorage('entity_view_display')->load($entity_type . '.' . $bundle . '.' . $mode)) {
```

Comment:
Thanks, that might be it - I'll check

we do try to set `entity_reference_label` as the formatter in `\Drupal\og\Plugin\OgFields\AudienceField::getViewDisplayDefinition` - but you gave me a direction :)